src/server/index.js

```diff
@@ -5,7 +5,40 @@ const config = require('./config.json')
 const app = express()
 const logger = require('./logger.js')
 
+const passport = require('passport')
+const expressSession = require('express-session')
+const MongoSessionStore = require('connect-mongo')(expressSession)
+
+const dbConnection = require('./dbConnection')
+const passportStrategies = require('./passport/strategies')
+const passportStrategyUtils = require('./passport/strategyUtils')
+const LocalStrategy = require('passport-local').Strategy
+
 app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: false }))
+
+app.use(expressSession({ 
+	store: new MongoSessionStore({ mongooseConnection: dbConnection} ),
+	secret: config.session.secret, 
+	resave: config.session.resave, 
+	saveUninitialized: config.session.saveUninitialized,
+	cookie: { 
+		secure: config.session.cookie.secure,
+		expires: new Date(Date.now() + config.session.cookie.expires),
+		maxAge: new Date(Date.now() + config.session.cookie.expires)
+	} }))
+
+app.use(passport.initialize())
+app.use(passport.session())
+
+const loginStrategy = passportStrategies.loginStrategy()
+passport.serializeUser(loginStrategy.serialzeUser)
+passport.deserializeUser(loginStrategy.deserializeUser)
+
+passport.use(passportStrategyUtils.STRATEGY_NAME, new LocalStrategy({
+	usernameField: passportStrategyUtils.FIELDS_NAMES.username_field,
```

Comment:
`username_field`?

Reply:
Strange convention. If that's constant, use uppercase.

Reply:
ok